api_core/google/api_core/grpc_helpers.py

```diff
@@ -136,3 +138,178 @@ def create_channel(target, credentials=None, scopes=None, **kwargs):
 
     return google.auth.transport.grpc.secure_authorized_channel(
         credentials, request, target, **kwargs)
+
+
+_ChannelCall = collections.namedtuple(
+    '_ChannelCall', ['request', 'timeout', 'metadata', 'credentials'])
+
+
+class _CallableStub(object):
+    """Stub for the grpc.*MultiCallable interfaces."""
+
+    def __init__(self, method, channel):
+        self._method = method
+        self._channel = channel
+        self.response = None
```

Comment:
Should `responses` alone be considered sufficient?
What happens if both are set?

Reply:
`response` will be ignored as it'll be overwritten with the first item in `responses`.